src/items.cpp

```diff
@@ -1188,6 +1188,9 @@ void Items::parseItemNode(const pugi::xml_node& itemNode, uint16_t id)
 						uint32_t ticks = 0;
 						int32_t start = 0;
 						int32_t count = 1;
+						int32_t initdamage = 0;
+						bool initdamageConfigured = false;
```

Comment:
this bool seeems redundant? You can simply check initDamage variable if its not 0
also rename initdamage to initDamage

Reply:
No because initDamage 0 does have a purpose. (to let us know that we don't want damage to override initdamage). 
Yes I can rename it.